trading/trader/trader.py

```diff
@@ -237,6 +243,35 @@ def create_order_instance(self, order_type, symbol, current_price, quantity,
             if order.get_status() is not OrderStatus.CANCELED:
                 await self.notify_order_close(order, True)
 
+    async def sell_everything(self, symbol, inverted):
+        created_orders = []
+        order_type = TraderOrderType.BUY_MARKET if inverted else TraderOrderType.SELL_MARKET
+        async with self.portfolio.get_lock():
+            portfolio = self.portfolio
+            current_symbol_holding, current_market_quantity, market_quantity, price, symbol_market = \
+                await AbstractTradingModeCreator.get_pre_order_data(self.exchange, symbol, portfolio)
+            quantity = current_market_quantity/price if inverted else current_symbol_holding
+            for order_quantity, order_price in AbstractTradingModeCreator.\
+                    check_and_adapt_order_details_if_necessary(quantity, price, symbol_market):
+                current_order = self.create_order_instance(order_type=order_type,
+                                                           symbol=symbol,
+                                                           current_price=order_price,
+                                                           quantity=order_quantity,
+                                                           price=order_price)
+                await self.create_order(current_order, portfolio)
```

Comment:
created_orders.append(`await self.create_order(current_order, portfolio)`)
And remove the following line. Because create_order also returns the order instance.